vividus-plugin-web-app/src/main/resources/properties/deprecated/deprecated.properties

```diff
@@ -1,4 +1,4 @@
-web\\.driver\\.edge_chromium.driver-executable-path=web.driver.edge.driver-executable-path
+web.driver.edge_chromium.driver-executable-path=web.driver.edge.driver-executable-path
```

Comment:
why is changed?

Reply:
> why is changed?

Character escaping works only with dynamic deprecated properties, in other cases it breaks deprecation of "usual" properties